roles/custom/matrix-base/templates/static-files/well-known/matrix-client.j2

```diff
@@ -26,6 +26,11 @@
 		"preferredDomain": {{ matrix_client_element_jitsi_preferredDomain|to_json }}
 	}
 	{% endif %}
+	{% if matrix_sliding_sync_proxy_enabled %},
+	"org.matrix.msc3575.proxy": {
+		"url": "https://{{ matrix_server_fqn_sliding_sync_proxy }}/_matrix/client/unstable/org.matrix.msc3575/sync"
```

Comment:
I believe this should not include the path to the sync API, just the URL of sync proxy:
```json
{
    "org.matrix.msc3575.proxy": {
        "url": "https://{{ matrix_server_fqn_sliding_sync_proxy }}"
    }
}
```

See Matrix.org's client well-known as an example:
https://matrix.org/.well-known/matrix/client

Reply:
While, I'd generally agree, the way I read the [Section in the MSC3575](https://github.com/matrix-org/matrix-spec-proposals/blob/kegan/sync-v3/proposals/3575-sync.md#unstable-prefix) as it is now, it specifically talks about the URL the sync is to be found at. I'll ask there for clarification.

Reply:
Hmm, the way I read that section was that both the well-known JSON key: `org.matrix.msc3575.proxy` and the path to the sync endpoint are prefixed.
One stabilized, the JSON key would just be `proxy`

Reply:
The Proxy is a temporary measure im not sure it will actually become stable. As in its likely there wont be a proxy that is under a stable identifier as this will just become sync v4 endpoint wise

Reply:
Clarification from Travis on the matter of the destiny of the proxy. It seems like its not going to become stabilised also as for what the /.well-known content should be to me it sounds like it should be 
```
{
    "org.matrix.msc3575.proxy": {
        "url": "https://{{ matrix_server_fqn_sliding_sync_proxy }}"
    }
}
``` 
as @Kab1r is suggesting.